api4/channel.go

```diff
@@ -1513,6 +1549,27 @@ func addChannelMember(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	categoryId, categoryIdOK := props["category_id"].(string)
```

Comment:
Nit: It's idiomatic to just use `ok`. 

Also, it might be cleaner to do an early return of `!ok || (categoryId != "" && !model.IsValidId(categoryId))`, rather than check for `ok` every time.